data_collection/gazette/spiders/sp_araraquara.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+from urllib.parse import urlencode
+
+import dateparser
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class SpAraraquaraSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3503208"
+    name = "sp_araraquara"
+    allowed_domains = ["diariooficialcmararaquara.sp.gov.br"]
+    start_date = datetime.date(2021, 3, 4)  # First gazette available
+    end_date = datetime.datetime.today()
```

Comment:
**suggestion** `end_date` value is defined as today in `BaseGazetteSpider` [definition](https://github.com/okfn-brasil/querido-diario/blob/main/data_collection/gazette/spiders/base/__init__.py#L38) so you don't need to specify it in your spider.